_posts/it/2022-01-25-Community-Call-importanza-design-system-servizi-pubblici digitali.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "Community Call: L’importanza di un design system per i servizi pubblici digitali"
+subtitle: Quali le sfide da affrontare? Ne parliamo il 9 febbraio!
+date: 2022-01-25 12:00:00 +0200
+tags: developers designers designsystem
+categories: news
+author: Michela Gabrieli
+image: /assets/icons/logo-it.png
+locale: it
+---
+
+Vi siete mai chiesti perché alcuni siti sono incredibilmente facili da navigare, e ci fanno sentire a casa fin dal primo utilizzo. Per esempio i grandi e-commerce, i social network, il nostro sito preferito di streaming? Ciò avviene perché tutte le grandi aziende digitali definiscono librerie,  componenti in uno stile unitario e regole condivise - il *design system* -  in modo che questi semplici “mattoncini” possano essere ripetuti e assemblati, creando per gli utenti un’esperienza coerente e soddisfacente. 
+
+Non sarebbe bellissimo se ciò accadesse anche per tutti i siti e i servizi digitali della Pubblica Amministrazione?
+
+Proprio per rispondere a questo quesito, a partire dallo scorso dicembre [Designers Italia](https://designers.italia.it/) in collaborazione con Developers Italia ha avviato una revisione completa del design system del Paese, uno strumento utile a  creare siti e servizi coerenti, familiari e fruibili da tutti i cittadini. A breve sarà resa pubblica una roadmap di aggiornamento delle librerie dei componenti e dei [due modelli applicati](https://designers.italia.it/modelli/), [Comuni](https://designers.italia.it/modello/comuni/) e [scuole](https://designers.italia.it/modello/scuole/). Questa revisione affronta una serie di sfide peculiari, data la sua applicazione in ambito pubblico:
```

Comment:
> in collaborazione con Developers Italia

Do we really want to keep Developers & Designers apart? Sounds too cold and formal to me